### PR TITLE
Feat/same sol confirm details

### PR DESCRIPTION
**Pull request by bhssamray**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: e2e/config.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ module.exports = {
   },
   secondDefendantSolicitorUser: {
     password: defaultPassword,
-    email: 'hmcts.civil+organisation.2.solicitor.2@gmail.com',
+    email: 'hmcts.civil+organisation.3.solicitor.1@gmail.com',
```
Comment (akriti-hmcts) [inline e2e/config.js:32]: why have you changed the user email?

File: e2e/fixtures/events/createClaim.js [hunk 2]
```diff
@@ -170,105 +178,268 @@ const createClaimData = (legalRepresentation, useValidPba) => {
...
+  //           ]
+  //         }
+  //       },
+  //       Court: {
+  //         courtLocation: {
+  //           applicantPreferredCourt: ['3a3', '21', '3333']
+  //         }
+  //       }
+  //     }
+  //   },
```
Comment (akriti-hmcts) [inline e2e/fixtures/events/createClaim.js:364]: can the commented code be removed?

**Review by akriti-hmcts — DISMISSED** [review]

**Review by akriti-hmcts — APPROVED** [review]

File: e2e/fixtures/events/createClaim.js (hunk 2 above)
Comment (bhssamray) [reply e2e/fixtures/events/createClaim.js]: Reverted unnecessary changes

File: e2e/config.js (hunk 1 above)
Comment (bhssamray) [reply e2e/config.js]: Reverted unnecessary changes

**Review by douglasrice — APPROVED** [review]

**Pull request merged**